src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -103,6 +109,102 @@ export class PatientViewPageStore {
         ))
     },{});
 
+    readonly geneticProfilesInStudy = remoteData(() => client.getAllGeneticProfilesInStudyUsingGET({
+        studyId: this.studyId
+    }), []);
+
+    private readonly mrnaRankGeneticProfileId = remoteData({
+        await: () => [
+            this.geneticProfilesInStudy
+        ],
+        invoke: () => {
+            const regex1 = /^.+rna_seq.*_zscores$/;
+            const regex2 = /^.*_zscores$/;
+            const chosenProfile: GeneticProfile = this.geneticProfilesInStudy.result.reduce((curr: GeneticProfile, next: GeneticProfile) => {

```

Comment:
alright i think maybe i understand.  if there's no match for regex1, then it will fall back to regex2?